### DISCUSSION TITLE
[Question]How to make text not wrap and overflow hidden ?

### DISCUSSION THREAD

**Discussion by yoyo837** [exceljs/exceljs#2490] [Q&A]

**Comment by andresmatasuarez** [comment]
+1

**Comment by wissam-natex** [comment]
+1

**Comment by NoxRammus** [comment]
You can get this working if you set wrapText to true on the cell, and then after setting that you set a max height on the row that cell was in.
The default row height in excel is 15.

E.g.: 
```javascript
row.getCell(1).alignment = { wrapText: true };
row.height = 15;
```
